bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/discovery/HueBridgeUPNPDiscoveryParticipant.java

```diff
@@ -0,0 +1,170 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link HueBridgeUPNPDiscoveryParticipant} is responsible for discovering new and removed Hue Bridges. It uses the
+ * central {@link UpnpDiscoveryService}.
+ *
+ * The discovery through UPnP was replaced by mDNS discovery for recent bridges (V2).
+ * For old bridges (V1), the UPnP discovery is still required (as mDNS is not implemented).
+ * This class allows discovering only old bridges using UPnP.
+ *
+ * @author Laurent Garnier - Initial contribution
+ */
+@Component(configurationPid = "discovery.hue")
+@NonNullByDefault
+public class HueBridgeUPNPDiscoveryParticipant implements UpnpDiscoveryParticipant {
+
+    private static final String EXPECTED_MODEL_NAME_PREFIX = "Philips hue bridge";
+
+    private final Logger logger = LoggerFactory.getLogger(HueBridgeUPNPDiscoveryParticipant.class);
+
+    private long removalGracePeriod = 50L;
+
+    private boolean isAutoDiscoveryEnabled = true;
+
+    @Activate
+    protected void activate(ComponentContext componentContext) {
+        activateOrModifyService(componentContext);
+    }
+
+    @Modified
+    protected void modified(ComponentContext componentContext) {
+        activateOrModifyService(componentContext);
+    }
+
+    private void activateOrModifyService(ComponentContext componentContext) {
+        Dictionary<String, @Nullable Object> properties = componentContext.getProperties();
+        String autoDiscoveryPropertyValue = (String) properties
+                .get(DiscoveryService.CONFIG_PROPERTY_BACKGROUND_DISCOVERY);
+        if (autoDiscoveryPropertyValue != null && !autoDiscoveryPropertyValue.isBlank()) {
+            isAutoDiscoveryEnabled = Boolean.valueOf(autoDiscoveryPropertyValue);
+        }
+        String removalGracePeriodPropertyValue = (String) properties.get(REMOVAL_GRACE_PERIOD);
+        if (removalGracePeriodPropertyValue != null && !removalGracePeriodPropertyValue.isBlank()) {
+            try {
+                removalGracePeriod = Long.parseLong(removalGracePeriodPropertyValue);
+            } catch (NumberFormatException e) {
+                logger.warn("Configuration property '{}' has invalid value: {}", REMOVAL_GRACE_PERIOD,
+                        removalGracePeriodPropertyValue);
+            }
+        }
+    }
+
+    @Override
+    public Set<ThingTypeUID> getSupportedThingTypeUIDs() {
+        return HueBridgeHandler.SUPPORTED_THING_TYPES;
+    }
+
+    @Override
+    public @Nullable DiscoveryResult createResult(RemoteDevice device) {
+        if (!isAutoDiscoveryEnabled) {
+            return null;
+        }
+        DeviceDetails details = device.getDetails();
+        ThingUID uid = getThingUID(device);
+        if (details == null || uid == null) {
+            return null;
+        }
+        URL baseUrl = details.getBaseURL();
+        String serialNumber = details.getSerialNumber();
+        String label = details.getFriendlyName();
```

Comment:
Perhaps this should be moved down and changed to:

```java
String label = String.format(DISCOVERY_LABEL_PATTERN, baseUrl.getHost());
```

for being consistent with #14919, WDYT? What will be the current value of this label?

Reply:
"Philips hue (IP)"
So the difference is only the h of hue that is in lowercase. 
I am ok to rather use DISCOVERY_LABEL_PATTERN.

Reply:
It would only be a problem if NUPnP would also work for the old bridge, otherwise there is only one discovery method, so no collisions like we had for mDNS and NUPnP. However, I guess consistency still wouldn't hurt and it would be safer - for example if NUPnP would start to work again suddenly.

Reply:
Yes

Reply:
Done (and tested).